environment/src/test/java/com/sequenceiq/environment/environment/flow/deletion/handler/ResourceEncryptionDeleteHandlerTest.java

```diff
@@ -50,6 +57,9 @@
     @Mock
     private Event<EnvironmentDeletionDto> environmentDtoEvent;
 
+    @Mock
+    private EnvironmentEncryptionService environmentEncryptionService;
```

Comment:
This is not used anywhere below. Please add at least one test method that verifies that `environmentEncryptionService.deleteEncryptionResources()` gets invoked when needed, and another that verifies that that method does not get invoked at all (if there is no DES to delete).